src/pe_reports/data/db_query.py

```diff
@@ -61,11 +61,11 @@ def get_orgs(conn):
             close(conn)
 
 
-def query_hibp_view(org_uid, start_date, end_date):
-    """Query 'Have I Been Pwned?' table."""
+def query_creds_view(org_uid, start_date, end_date):
+    """Query credentials view ."""
```

Comment:
Remove errant extra space:
```suggestion
    """Query credentials view."""
```